openpype/hosts/resolve/plugins/load/load_media.py

```diff
@@ -0,0 +1,177 @@
+import json
+
+from openpype.client import (
+    get_version_by_id,
+    get_last_version_by_subset_id,
+)
+from openpype.pipeline import (
+    LoaderPlugin,
+    get_representation_path
+)
+from openpype.pipeline.context_tools import get_current_project_name
+from openpype.hosts.resolve.api import lib
+from openpype.hosts.resolve.api.pipeline import AVALON_CONTAINER_ID
+from openpype.lib.transcoding import (
+    VIDEO_EXTENSIONS,
+    IMAGE_EXTENSIONS
+)
+
+
+class LoadMedia(LoaderPlugin):
+    """Load a subset as media pool item."""
+
+    families = ["render2d", "source", "plate", "render", "review"]
+
+    representations = ["*"]
+    extensions = set(
+        ext.lstrip(".") for ext in IMAGE_EXTENSIONS.union(VIDEO_EXTENSIONS)
+    )
+
+    label = "Load media"
+    order = -10
+    icon = "code-fork"
+    color = "orange"
+
+    # for loader multiselection
+    timeline = None
+
+    # presets
+    clip_color_last = "Olive"
+    clip_color = "Orange"
+
+    bin_path = "Loader/{representation[context][hierarchy]}/{asset[name]}"
+
+    def load(self, context, name, namespace, options):
+
+        # For loading multiselection, we store timeline before first load
+        # because the current timeline can change with the imported media.
+        if self.timeline is None:
+            self.timeline = lib.get_current_timeline()
+
+        representation = context["representation"]
+
+        project = lib.get_current_project()
+        media_pool = project.GetMediaPool()
+
+        # Create or set the bin folder, we add it in there
+        # If bin path is not set we just add into the current active bin
+        if self.bin_path:
+            bin_path = self.bin_path.format(**context)
+            lib.create_bin(
+                name=bin_path,
+                root=media_pool.GetRootFolder()
+            )
+
+        # TODO: Correctly set the media path for image sequences
+        # frames = [
+        #     "path/to/frame.1001.exr",
+        #     "path/to/frame.1002.exr"
+        # ]
+        path = get_representation_path(representation)
+        items = media_pool.ImportMedia(path)
```

Comment:
Yes, this is a great approach. We should definitely fix this before the merge. I can help ;)